install-dev/data/db_structure.sql

```diff
@@ -1854,6 +1854,7 @@ CREATE TABLE `PREFIX_tab` (
   `position` int(10) unsigned NOT NULL,
   `active` tinyint(1) NOT NULL DEFAULT 1,
   `hide_host_mode` tinyint(1) NOT NULL DEFAULT '0',
+  `icon` varchar(32) DEFAULT '',
```

Comment:
When the final navigation will be finished (maybe some things will move), the upgrade will be done for this, and all the modifications done on the xml files.